Runtime/Shaders/meshsync_channel_mapping.compute

```diff
@@ -0,0 +1,45 @@
+#pragma kernel smoothness_into_alpha
+#pragma kernel hdrp_mask
+
+// Create a RenderTexture with enableRandomWrite flag and set it
+// with cs.SetTexture
+RWTexture2D<float4> Output;
+
+Texture2D<float3> RGB;
+Texture2D<float> Smoothness;
+Texture2D<float> Metallic;
+
+SamplerState samplerRGB;
+SamplerState samplerSmoothness;
+SamplerState samplerMetallic;
+
+[numthreads(8, 8, 1)]
+void smoothness_into_alpha(uint3 id : SV_DispatchThreadID)
+{
+    int2 rgbDimensions;
+    RGB.GetDimensions(rgbDimensions.x, rgbDimensions.y);
```

Comment:
We don't know the size. This could be a texture we created from a constant value or a texture the user set. The GetDimensions call is not slow, the baking is very fast.

Reply:
This shader is executed once, I don't think it's worth the added complexity and chance to introduce bugs that happen when the texture size changes.

Reply:
It's just setting a constant buffer at the same time as setting the texture, and a better practice in general.
I don't think it's that complex, is it ?

Reply:
So you'd say it's better practice to set the texture and the texture size and potentially not setting the texture size when it's called from another place instead of getting the texture size in the shader from the texture to ensure it's up to date?

Reply:
Yes, due to the following reasons:

1. it's faster
2. it's not just anywhere we can generally call SetTexture() to a computer shader anyway, and when we do that, we generally should be certain if it's intended.
3. if we wrap the SetTexture() and SetConstantBuffer() in a function, I think the chance is very low to have this problem.

Reply:
I've changed it now.